tests/network/hotplug.go

```diff
@@ -238,6 +238,29 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
+
+		DescribeTable("VMI status should report the MAC address specified in the VM template spec", func(plugMethod hotplugMethod) {
+			waitForSingleHotPlugIfaceOnVMISpec(hotPluggedVMI)
+			hotPluggedVMI = verifyDynamicInterfaceChange(hotPluggedVMI, plugMethod)
+
+			var err error
+			hotPluggedVM, err := kubevirt.Client().VirtualMachine(hotPluggedVM.Namespace).Get(context.Background(), hotPluggedVM.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			vmIfaceSpec := vmispec.LookupInterfaceByName(hotPluggedVM.Spec.Template.Spec.Domain.Devices.Interfaces, ifaceName)
+			Expect(vmIfaceSpec).NotTo(BeNil(), "VM spec should contain the new interface")
+			Expect(vmIfaceSpec.MacAddress).ToNot(BeEmpty(), "iface MAC address in VM spec should not be empty")
+
+			vmiIfaceStatus := vmispec.LookupInterfaceStatusByName(hotPluggedVMI.Status.Interfaces, ifaceName)
```

Comment:
I would add Eventually here (next 4 lines) to avoid flakiness. It may take time for the MAC be reported by the GA.

Reply:
Done